index.html

```diff
@@ -1,158 +1,282 @@
+            <ul class="technologies">
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>Javascript</li>
             </ul>
+            <button class="button">See this project<span class="material-symbols-outlined"> arrow_forward </span>
+            </button>
           </div>
         </div>
-      </section>
-    </main>
-  </body>
+      </div>
+    </section>
+
+    <section class="spacing">
+      <div>
+        <h2 class="title">About</h2>
+        <p class="paragraph-headline">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Inventore
+          iste laboriosam atque numquam quod dolorum aspernatur amet, cupiditate et ratione expedita veritatis
+          necessitatibus alias in.
+        </p>
+        <a class="link-headline link-icon" href="" download="">Get my CV<span class="material-symbols-outlined">
+            download </span>
+        </a>
+      </div>
+      <div>
+        <div class="card bgc1">
+          <h3 class="about-cards">Languages</h3>
+          <ul class="about-info">
+            <li>Javascript (ES5/ES6)</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+        <div class="card bgc2">
+          <h3 class="about-cards">Frameworks</h3>
+          <ul class="about-info">
+            <li>Bootstrap</li>
+            <li>Ruby on rails</li>
+            <li>RSpec</li>
+            <li>Capibara</li>
+            <li>Selenium</li>
+          </ul>
+        </div>
+        <div class="card bgc3">
+          <h3 class="about-cards">Skills</h3>
+          <ul class="about-info">
+            <li>Codekit</li>
+            <li>Codepen</li>
+            <li>GitHub</li>
+            <li>GitLab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      </div>
+    </section>
+  </main>
+  <footer>
+    <section>
+      <div class="bkg spacing">
+        <h2 class="title stylefooter">Get started</h2>
+        <p class="paragraph-headline">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+          Natus magni minus nesciunt id corporis accusamus.
+        </p>
+        <form class="form" action="" method="post">
+          <ul class="footercontainer">
+            <li>
+              <label for=""></label>
+              <input class="formli" type="text" id="name" name="user_name" placeholder="Your name" maxlength="30" required/>
+            </li>
+            <li>
+              <label for=""></label>
+              <input class="formli" type="email" id="mail" name="user_email" placeholder="Your email" required />
+            </li>
+            <li>
+              <label for=""></label>
+              <textarea class="formli textform" id="msg" name="user_message" placeholder="Enter your message here..." maxlength="500" required></textarea>
+            </li>
+          </ul>
+          <a class="link-headline footerstyle" href="">Start collaboration</a>
```

Comment:
- [ ] Kindly make sure the submit button can be either an `input` element with the type `submit` or a `button` element with the type `submit`. This will help to able to submit your form successfully.